src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -234,11 +270,25 @@ public DeploymentOptions()
             foreach (var deployOptionsProp in deploymentOptionsProperties)
             {
                 var prop = options.GetType().GetProperty(deployOptionsProp.Name);
+                var attribute = prop.GetCustomAttributes(typeof(DescriptionAttribute), true)[0];
```

Comment:
1. Is this safe to do? It would be best to assume this property won't always have the attribute on it and handle that correctly.
2. Would suggest using the generics version of this method (`GetCustomAttributes<DescriptionAttribute>(true)`)
3. Generally you should do `FirstOrDefault` instead of directly accessing the first element, especially when you can't be sure that the list isn't empty. 
4. You shouldn't need the cast to DescriptionAttribute with the proper typing on the GetCustomAttributes call